setup/src/App.svelte

```diff
@@ -5,25 +5,34 @@
   let domain = 'INSERT_DOMAIN_HERE';
   let auth: undefined | 'newApp' | 'existingApp' | 'pat';
   let appScope: 'user' | 'org' = 'user';
+  let runnerLevel: 'repo' | 'org';
   let org = 'ORGANIZATION';
   let existingAppId: string = '';
   let existingAppPk: string = '';
   let pat: string = '';
   let success: boolean;
   let result: string | undefined;
 
+  const repositoryPermissions = {
+    actions: 'write',
+    administration: 'write',
+    deployments: 'read',
+  };
+
+  const organizationPermissions = {
+    actions: 'write',
+    organization_self_hosted_runners: 'write',
+    deployments: 'read',
+  };
+
   const manifest = {
     url: 'https://github.com/CloudSnorkel/cdk-github-runners',
     hook_attributes: {
       url: 'INSERT_WEBHOOK_URL_HERE',
     },
     redirect_url: 'INSERT_BASE_URL_HERE/complete-new-app',
     public: false,
-    default_permissions: {
-      actions: 'write',
-      administration: 'write',
-      deployments: 'read',
-    },
+    default_permissions: repositoryPermissions || organizationPermissions,
```

Comment:
Why `||` here? Won't that always be `repositoryPermissions` because they're both truthy? Is that to get TypeScript off your back?

Reply:
Yes had some issues with the linter it seems. Makes sense to set repositorypermissions as default. Will try to change it.

Reply:
Checked it again ... the reason is a typescript error ... 
```bash
Property 'administration' is missing in type '{ actions: string; organization_self_hosted_runners: string; deployments: string; }' but required in type '{ actions: string; administration: string; deployments: string; }'.ts(2322)
```